### PR TITLE
Remove libcellml refs

### PR DESCRIPTION
**Pull request by kerimoyle**
Removes references to libCellML that aren't needed, addresses [this comment](https://github.com/cellml/cellml-specification/pull/99#discussion_r399035193).

### CODE REVIEW

**Comment by kerimoyle** [comment]
Bother, wrong branch ...

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cellml-specification
 ====================
 
-This repository will be used by the CellML editors to develop the CellML specification (starting with version 1.2). The content was initially obtained from https://github.com/A1kmm/cellml-core-spec/tree/normative, using pandoc to convert the docbook XML to reST format (``for f in *.xml; do pandoc -f docbook -o `basename $f .xml`.rst $f; done``). The reST documents were then incorporated into a basic sphinx project.
+This repository will be used by the CellML editors to develop the CellML specification (starting with version 1.2). The content has been reworked to include an informal guide embedded within the normative specification text (in version 2.0).
```
Comment (MichaelClerx) [inline README.md]: Version 1.2????

File: conf.py [hunk 2]
```diff
@@ -42,18 +50,84 @@
...
+# rst_epilog = """
+# .. |component| replace:: ``component``
+# .. _component: :ref:`component`
+# """
+
 # General information about the project.
-project = u'CellML 1.2 Specification'
-copyright = u'2013, The CellML Editorial Board'
+project = u'CellML 2.0'
+copyright = u'2019-@SPHINX_YEAR@, CellML Editors and Contributors'
```
Comment (MichaelClerx) [inline conf.py]: Should this not be 2013-@SPHINX_YEAR@ ?

File: README.md (hunk 1 above)
Comment (agarny) [reply README.md]: Also two sentences = two lines. :)

**Pull request closed without merging**